src/coord/src/coord.rs

```diff
@@ -2202,6 +2202,42 @@ impl Coordinator {
         })
     }
 
+    /// Return the set of ids in a timedomain and verify timeline correctness.
+    ///
+    /// When a user starts a transaction, we need to prevent compaction of anything
+    /// they might read from. We use a heuristic of "anything in the same database
+    /// schemas with the same timeline as whatever the first query is".
+    fn timedomain_for(
```

Comment:
I feel like we don't need the word `timedomain` anymore -- like this function might just be called `usable_objects_for` or something like that and the field in TxnReads could be called `usable_objects` or `usable_ids` -- does that sound right?

Reply:
The concept of timedomains was discussed in #4660 and we probably need to increase documentation about it, and want to keep the name because we might allow user-defined timedomains in the future.

Reply:
ah ok I didnt have all of that context sounds good!